### DISCUSSION TITLE
Can I access my Birdnet running on a PI off site? Like remote access?

### DISCUSSION THREAD

**Discussion by Billium28** [mcguirepr89/BirdNET-Pi#820] [General]
Is there a way to log in to the ui from the web?

**Comment by ehpersonal38** [comment]
https://github.com/mcguirepr89/BirdNET-Pi/wiki/Sharing-Your-BirdNET-Pi#configure-port-forwarding

The basic idea is that you'll need to configure port forwarding on your router for port 80. Then you'll need to lookup your public IP address (Google it and it'll show), and then you'll be able to access your BirdNET-Pi by navigating to http://{your public ip address} .

There are a ton of resources for port forwarding online you can lookup. It can be a really tedious process so don't worry if it takes a while to get setup. For more resources I recommend searching something like "how to port forward on {your router model}."

**Comment by ampsu** [comment]
If you can not do any port forwarding or are using a LTE network. You can use some HTTP tunneling service. I am using SocketXP and Ngrok for remote battery powered Pi-s that are far away in the forest. Both work fine and quite easy to setup. You can set them up as service so they would automatically start when you power the Pi up.

**Reply by ehpersonal38** [reply]
That does look good, though it's only free for a 30 day trial. Here's another VPN tunneling service which is free for personal use: https://tailscale.com/.

And actually if you can get this working it's probably better than port forwarding.

**Reply by ampsu** [reply]
Ngrok is free, SocketXP is paid but comes with 30 day trial. I have not tried Tailscale but I will also look into it. Looks like they support 20 devices for free which great for many Pi-s.

**Reply by oystercatcher-181** [reply]
Wondered what you thought of tailscale to access birdnet-pi remotely on another network to view detections and listen to recording etc?

I am trying to find a solution to gaining access to a couple of birdnet-pi’s in different locations (at my parents house etc.) without leaving thier network open the dangers of port forwarding? Any advice?

**Reply by tphakala** [reply]
I am using cloudflared with my birdnet-go, it is working very well. Haven't tested it with birdnet-pi but I guess it should work with it also https://github.com/cloudflare/cloudflared

**Reply by nickds1** [reply]
I use port forwarding on the house router. As we don't have a static IP address for our internet connection, I use a free dynamic DNS service to give a name to whatever IP address we happen to have, e.g. my-birdnet-pi@dyndns.com (several of these free services are available - use whatever ones your router supports).

I then have a CNAME record for one of our "in house" family domain names, birdnet.desmith.something that points to the rather obtuse dyndns one, so friends & family can use a familiar name (literally) to access our birdnet.

Works perfectly. And is free!

**Reply by oystercatcher-181** [reply]
I just wanted to reply to say that thanks for the guidance, I am more of a birder than a coder and this information really helped us. In the end, I went for Cloudflare Tunnel on my BirdNet-Pi and used a couple of videos on YouTube to go through the details. Our research team have now got 4 birdnet-pi's accessible in Scotland, England and soon West Africa. Thanks for helping our research!

**Comment by Fryerchristopher** [comment]
I grabbed a domain on GoDaddy to access my BirdNetPi publicly.  As it is setup in a location with a dynamic IP, I used the following website to create a script that checks the local WAN IP and then adjusts the host A record automatically at GoDaddy.  I'm a weak programmer/scripter so it took a bit of messing around but it is working.

https://www.instructables.com/Quick-and-Dirty-Dynamic-DNS-Using-GoDaddy/
